src/retasc/validator/models.py

```diff
@@ -0,0 +1,22 @@
+from pydantic import BaseModel
+
+
+class Subtask(BaseModel):
+    template: str
+
+
+class JiraIssue(BaseModel):
+    template: str
+    subtasks: list[Subtask] | None = None
```

Comment:
I think the following is also possible and we could drop `Subtask`:

```suggestion
    subtasks: list[JiraIssue] = []
```

Reply:
I have modified it with Field and its default parameter